teorija.tex

```diff
@@ -4069,9 +4068,9 @@ \subsection{Krog}
 Razmerje med obsegom in premerom kroga je \textbf{konstantno}. Konstanto označimo s~$\boldsymbol\pi$.
 \begin{align*}
   o &= 2\pi r \\
-  l &= \frac{\pi r\alpha}{180\deg} = r\alpha \\
+  l &= \frac{\pi r\alpha}{180\deg} = r\theta \\
   p &= \pi r^2 \\
-  p_{iz} &= \frac{\pi r^2\alpha}{360\deg} = \frac{r^2\alpha}{2} = \frac{l\krat
+  p_{iz} &= \frac{\pi r^2\alpha}{360\deg} = \frac{r^2\theta}{2} = \frac{l\krat
```

Comment:
Hm, ja, to je vedno problematično, ker radian ni enota, radian je stevilo. In vsi koti so številke. Znak stopinja je **definiran** kot **stevilo** \frac{pi}{180}, tako da, ko ti napišeš 180 deg, se razume, da je to `180 * pi / 180 = pi` in torej v radianih. Če pač namesto `\deg` v formulao vstavis `pi/180` dobis desno stran. Se pa strinjam, da lahko dodas `\comm{$\alpha$ v radianih}`, pa bo bolj jasno.